.github/workflows/build.yml

```diff
@@ -0,0 +1,60 @@
+name: build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        rust: [stable, nightly]
+
+    steps:
+    - uses: hecrj/setup-rust-action@v1
```

Comment:
what does `hecrj/setup-rust-action` provide compared to `action-rs/toolchain`?
also, it would be nice to use cache, see https://github.com/iqlusioninc/crates/blob/develop/.github/workflows/rust.yml

Reply:
Mostly simplifies the install of the rustc version & components with
```yml
with:
  rust-version: stable
  components: clippy
```